docs/dataset-overviews/solar.md

```diff
@@ -0,0 +1,92 @@
+# Solar
+
+**All of the text below is a draft written by a non-expert to get things moving.**
+**Please do not put any faith in its accuracy.**
```

Comment:
@berndfunke @st-bender if you're able to take a look at this file, that would be great. I think I have captured all the information correctly and pushed people to your original data source quickly enough, but your double checking of this would be great.

Reply:
If you're interested, the rendered version of this text is at: https://input4mips-controlled-vocabularies-cvs--150.org.readthedocs.build/en/150/dataset-overviews/solar/